tools/kayobe-dev.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eux 
+
+source ~/venvs/kayobe/bin/activate
```

Comment:
These paths won't always be correct - people install to different (or multiple) locations, use other vault file names, etc.